android/settings.gradle

```diff
@@ -1,33 +1,7 @@
 rootProject.name = 'ZulipMobile'
-
+apply from: file("../node_modules/@react-native-community/cli-platform-android/native_modules.gradle"); applyNativeModulesSettingsGradle(settings)
```

Comment:
similarly, let's uncramp this onto two lines

Also the blank lines seem helpful, separating logical stanzas of related code.